src/rgw/driver/d4n/rgw_sal_d4n.cc

```diff
@@ -2247,7 +2220,7 @@ int D4NFilterObject::D4NFilterDeleteOp::delete_obj(const DoutPrefixProvider* dpp
       do { // loop through the data blocks
         std::string prefix = get_cache_block_prefix(source, version, false);
         if (fst >= lst) {
-    break;
+	  break;
```

Comment:
Same comment as below.

Reply:
Same as below, remove your edits if not required.